azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/action/Action.java

```diff
@@ -64,6 +64,7 @@
     private List<Function<D, String>> titleParamProviders = new ArrayList<>();
 
     private D source;
+    private String place;
```

Comment:
place should not be a property of action. because action can be placed anywhere.

Reply:
remove it in late PR, please.